source/adios2/engine/bp5/BP5Reader.cpp

```diff
@@ -299,7 +299,7 @@ void BP5Reader::PerformGets()
         if (getenv("DoXRootD"))
         {
             m_Remote = std::unique_ptr<XrootdRemote>(new XrootdRemote());
-            m_Remote->Open("localhost", 1049, m_Name, m_OpenMode, RowMajorOrdering);
+            m_Remote->Open("localhost", 1094, m_Name, m_OpenMode, RowMajorOrdering);
```

Comment:
[optionally] I assume this is the port, should we create a variable for it so it's clear what it is?

Reply:
Yes, there are a number of things like that that I'm assuming that we'll get from the campaign-level stuff, dealing with multiple hosts and multiple configurations.  For now just hardcoding those things until I get the actual functionality in shape.